vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,139 +2,77 @@
-
-        npm install && npm start
-
-3.  When the widget starts, a browser window opens the widget URL
-
-4.  If you’re not logged in, you'll be redirected to the login page.
-
-5.  Log in using:
-
-        Username: user
-        Password: user
-
-6.  Once logged in, you can see the table widget with generated data.
-
-### Start the Form Widget
-
-1.  If you are already running a widget, click `Ctrl+C` to end the process
-
-2.  Go to the form widget folder in your project:
-
-        cd ui/widgets/YOUR-ENTITY-NAME/formWidget
-
-3.  Install and start your widget with the command:
-
-        npm install && npm start
-
-4.  When the widget starts, a browser window opens the widget URL 
-
-5.  If you’re not logged in, you'll be redirected to the login page
-
-6.  Log in using:
 
-        Username: user
-        Password: user
+### Run the conference-form MFE
 
-7.  Once logged in, you will see the widget form.
+1.  If you are still running the previous micro frontend, click `Ctrl+C` to end the process
 
-#### Form Widget notes:
+2. Start up the conference-form MFE 
+``` sh
+ent bundle run conference-form
+```
+The form to enter the name and location for conferences shoud open in your browser. You may enter and save new data to see it in the other MFEs.
 
-If you want to modify a different row in the database, edit the `index.html` file in the public folder `ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public`. Change the `id` attribute in this line:
+>If you want to modify a different row in the database, edit `microfrontends/conference-form/public/index.html` file. Change the `id` attribute in this line:
 ```
    <my-entity-form service-url="%REACT_APP_SERVICE_URL%" id="1" />
 ```
-### Start the Details Widget
-
-1. If you are already running a widget, click `Ctrl+C` to end the process
-
-2. Go to the details widget folder in your project:
+### Run the conference-details MFE
 
-        cd ui/widgets/YOUR-ENTITY-NAME/detailsWidget
+1. If you are running the previous micro frontend, click `Ctrl+C` to end the process
 
-3. Install and start your widget with the command:
-
-        npm install && npm start
-
-4. When the widget starts, a browser window opens with the widget URL
-
-5. If you’re not logged in, you'll be redirected to the login page
-
-6. Log in using:
-
-        Username: user
-        Password: user
-
-7.  Once logged in, you can see the widget details with the ID 1 loaded
-
-#### Widget Details notes:
+2. Start the conference-details MFE
+``` sh
+ent bundle run conference-details
+```
+When the run is complete, you should see the details MFE with the ID 1 loaded.
 
-If you want to modify a different row in the database, edit the `index.html` file in the public folder `ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public`. Change the `id` attribute in this line:
+>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
```

Comment:
@nshaw, same here, cannot find the id attribute

Reply:
Ditto, my bad